packet.c

```diff
@@ -2160,6 +2176,133 @@ ssh_packet_disconnect(struct ssh *ssh, const char *fmt,...)
 	cleanup_exit(255);
 }
 
+#ifdef MPTCP_GET_SUB_IDS
+
```

Comment:
Done in commit 906d3d6
But don't know if the struct also must be in the header file